.circleci/config.yml

```diff
@@ -19,18 +19,22 @@ jobs:
     steps:
       - checkout
 
+      # JS bundle
       - restore_cache:
           key: yarn-{{ checksum "yarn.lock" }}
 
       - run:
           name: Install yarn dependencies
-          command: yarn install
+          command: yarn install --frozen-lockfile
 
       - save_cache:
           key: yarn-{{ checksum "yarn.lock" }}
           paths:
+            - node_modules
+            - ~/.cache/yarn
             - ~/.yarn-cache
```

Comment:
So we need both `~/.cache/yarn` and `~/.yarn-cache`?

Reply:
just confirmed that we don't 👍